src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3:
+      return days * 40;
```

Comment:
create a variable for days * 40, because this calculation repeat a few times at your code